README.md

```diff
@@ -191,6 +191,7 @@ Replica Sets | Beta (1.2)<br>Beta (1.3) | Disabled Pending Migration (1.2)<br>Te
 Ingress | Alpha (1.1)<br>Beta (1.2, 1.3) | Disabled Pending Migration (1.2, 1.3) | OpenShift launched with Routes, a more full featured Ingress object. We plan to enable upstream Ingresses with automatic migrations to Routes so that existing clusters continue to function as normal without a migration, and so that existing client tools automatically display Ingresses.<br>Upstream ingress controllers are not supported, since the integrated router is production supported with a superset of Ingress functionality.
 PodSecurityPolicy | Alpha (1.2)<br>Beta (1.3) | Disabled Pending Migration (1.3)<br>Not Yet Secure (1.3) | OpenShift launched with SecurityContextConstraints, and then upstreamed them as PodSecurityPolicy. We plan to enable upstream PodSecurityPolicy so as to automatically migrate existing SecurityContextConstraints. PodSecurityPolicy has not yet completed a full security review, which will be part of the criteria for tech preview. <br>SecurityContextConstraints are a superset of PodSecurityPolicy features.
 PodAntiAffinitySelectors | Alpha (1.3) | Not Yet Secure (1.3)<br>Tech Preview (1.4?) | End users are not allowed to set PodAntiAffinitySelectors that are not the node name due to the possibility of attacking the scheduler via denial of service.|
+NetworkPolicy | Beta (1.3) | Disabled Pending Migration /<br> Tech Preview (1.3) | The default "OpenShift SDN" networking plugins (`redhat/openshift-ovs-subnet` and `redhat/openshift-ovs-multitenant`) do not support NetworkPolicy yet. However, OpenShift allows the use of third-party Kubernetes network plugins, which may support NetworkPolicy.
```

Comment:
So we explicitly disabled it?  If we didn't disable it but we don't support it that would be `Tech Preview` for other parties (because it's in beta).  Is there a plan to support NetworkPolicy instead of our existing APIs, and do we have a firm release (or just a desired release)?  If just desired, then I would recommend saying "Tech Preview" and note the limitation.  DisabledPendingMigration applies more to when we totally prevent the use of an API type, and in this case we just ignore it.